test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeModule.cs

```diff
@@ -92,23 +92,31 @@ public Task WaitForStatusAsync(EdgeModuleStatus desired, CancellationToken token
             return WaitForStatusAsync(new[] { this }, desired, token);
         }
 
-        public Task WaitForEventsReceivedAsync(DateTime seekTime, CancellationToken token)
+        public Task<string> WaitForEventsReceivedAsync(DateTime seekTime, CancellationToken token, params string[] requiredProperties)
```

Comment:
I don't use it at all since I switched to using a direct method. I left it since its useful if we want to validate the other tests responses. I can remove it if you want.

Reply:
No biggie either way...